### ISSUE TITLE
iOS File Uploading in Background 

### ISSUE THREAD

**Issue by MagneticLlama** [codemillmatt/CheesedStorage#1]
Am I correct in believing uploading cannot be persisted in the background on iOS with this solution? From what I understand, iOS requires we upload using an NSUrlSession if we want the upload to continue after the app is put in the background.

**Comment by codemillmatt** [comment]
That is correct - for demo purposes, I assumed the app would always be in the foreground.
